_includes/markdown/React.md

```diff
@@ -33,6 +33,25 @@ If you already have a Node / NPM / Yarn project that you wish to include React a
 
 Using the CLI tool is ideal if you are creating a full React App with an API backend.
 
+## Components
+When building out components, its beneficial to understand how to construct them in the most appropriate way possible. Certain "types" of components can be written differently which can have big performance benefits on larger scale applications.
```

Comment:
change its to it's
Can we also add some links to React documentation if people want to read more? Just linking up some words in the paragraph would suffice.